R/select_tables_from_dm.R

```diff
@@ -39,7 +37,8 @@ cdm_select_tbl <- function(dm, ..., all_connected = TRUE) {
     src = cdm_get_src(dm),
     tables = table_objs,
     data_model = new_data_model
-  )
+  ) %>%
+    {if (any(old_table_names != new_table_names)) {cdm_rename_tbl(., old_table_names)} else .}
```

Comment:
Don't need to special-case here:

```suggestion
    cdm_rename_tbl(old_table_names)
```